doc/dependencies.md

```diff
@@ -14,7 +14,7 @@ These are the dependencies currently used by Bitcoin Core. You can find instruct
 | GCC |  | [4.8+](https://gcc.gnu.org/) (C++11 support) |  |  |  |
 | HarfBuzz-NG |  |  |  |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk) |
 | libevent | [2.1.11-stable](https://github.com/libevent/libevent/releases) | [2.0.21](https://github.com/bitcoin/bitcoin/pull/18676) | No |  |  |
-| libpng |  |  |  |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk) |
+| libpng |  |  |  |  | No |
```

Comment:
Any reason you haven't filled in `Version used` and `CVEs` here? `CVE-2019-7317` applies to libpng prior to 1.6.37.